{{cookiecutter.__project_slug}}/action.json

```diff
@@ -5,5 +5,6 @@
         "vCPU": 512,
         "memory": 1024,
         "storage": 21
-    }
+    },
+    "parameters": []
```

Comment:
This is really hard to fill out from memory -- would be helpful if we had a jsonschema definition to give the editor autocomplete.

IIRC jsonschema just needs a URL reference in the top of the file, so we could host it in roboto-python-sdk, and refer to it by the roboto SDK release tag that was used to make the schema file.

_a la_ 
```jsonc
{
  "$schema": "http://github.com/roboto-ai/roboto-python-sdk/tags/v0.29.1/action-jsonschema.json",
  // ...the rest of the file
}
```